index.html

```diff
@@ -0,0 +1,477 @@
+                <li><a href="#"></a></li>
+              </ul>
+              <div class="next-arrow b-radius-10"></div>
+            </article>
+          </div>
+          <div class="article-div3 article-div">
+            <article class="article">
+              <div class="d-flex">
+                <div class="writer-img b-radius-10 rotate-15"></div>
+                <cite><a href="#">Lorem, ipsum.</a>wrote:</cite>
+              </div>
+              <h2 class="article-title">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                <i class="fas fa-comment-alt"></i>
+                <span>Lorem a dssllm</span>
+              </h2>
+              <ul class="d-flex article-hashtag width-100">
+                <li><a href="#"></a></li>
+                <li><a href="#"></a></li>
+                <li><a href="#"></a></li>
+              </ul>
+              <div class="next-arrow b-radius-10"></div>
+            </article>
+          </div>
+          <div class="article-div4 article-div">
+            <article class="article">
+              <div class="d-flex">
+                <div class="writer-img b-radius-10 rotate-15"></div>
+                <cite><a href="#">Lorem, ipsum.</a>wrote:</cite>
+              </div>
+              <h2 class="article-title">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                <i class="fas fa-comment-alt"></i>
+                <span>Lorem a dssllm</span>
+              </h2>
+              <ul class="d-flex article-hashtag width-100">
+                <li><a href="#"></a></li>
+                <li><a href="#"></a></li>
+                <li><a href="#"></a></li>
+              </ul>
+              <div class="next-arrow b-radius-10"></div>
+            </article>
+          </div>
+        </div>
+        <div class="centered-div">
+          <div class="b-radius-10"></div>
+          <div class="b-radius-10"></div>
+        </div>
+        <div class="switcher1 b-radius-10"></div>
+        <div class="switcher2 b-radius-10"></div>
+        <!-- MEMBERSHIP ANNOUNCEMENT -->
+        <div class="membership-container">
+          <div class="membership d-flex">
+            <div class="info b-radius-10"></div>
+            <div class="prom b-radius-10"></div>
+            <div class="prom b-radius-10"></div>
+            <div class="prom b-radius-10"></div>
+            <button type="button" class="button b-radius-20"></button>
+          </div>
+          <div class="membership-logo b-radius-20 rotate-15"></div>
+        </div>
+        <!-- ARTICLES -->
+        <div class="more-articles-div d-flex">
+          <div class="article1">
+            <div class="time b-radius-10"></div>
+            <div class="title b-radius-10"></div>
+            <div class="citing b-radius-10"></div>
+            <div class="paragraph b-radius-10"></div>
+            <div class="second-arrow b-radius-10"></div>
+          </div>
```

Comment:
- [ ] This is the main reason the project was not approved. All the articles should be covered by article tags.

![image](https://user-images.githubusercontent.com/61479648/91359603-4037d680-e7cb-11ea-82af-898ebdbadcf5.png)